DynMiner2/cStatDisplay.cpp

```diff
@@ -74,6 +75,40 @@ void cStatDisplay::displayStats(cSubmitter *submitter, string mode, int hiveos)
 
             std::string uptime = seconds_to_uptime(difftime(now, start));
 
+	    if (!statURL.empty()) {
+
+	        curl = curl_easy_init();
```

Comment:
This is a bit of a nitpick, but I feel like this should be a local variable for this function, not a class variable, because it's only used in this one block, and it'd be easy for somebody to decide to initialize it in a constructor and then have this function overwrite and close it.

Reply:
Fixed as well per your suggestion.